src/components/InputTooltip.js

```diff
@@ -182,15 +251,19 @@ class InputTooltip extends Component {
                             />
                         )}
                         <WdsIconsCheckmarkSmall
-                            onClick={() => this.accept(linkValue)}
+                            onClick={() => this.accept(linkHref)}
                             className="wds-icon wds-icon-small pe-input-tooltip__accept"
                         />
                     </div>
-                    {/* fixme error message should be different for isError = true */}
-                    {(isLinkInvalid || isError) && <span className="wds-input__hint">{i18n['hyperlinking-error']}</span>}
+                    {isLinkInvalid && <span className="wds-input__hint">{i18n['hyperlinking-error']}</span>}
                 </div>
-                {linkValue && (
-                    <div className="pe-input-tooltip__suggestions wds-dropdown__content wds-is-not-scrollable">
+                {linkHref && (
+                    <div
+                        className="pe-input-tooltip__suggestions wds-dropdown__content wds-is-not-scrollable"
+                        role="presentation"
+                        onMouseDown={this.setSuggestionsListClicked.bind(this, true)}
+                        onMouseUp={this.setSuggestionsListClicked.bind(this, false)}
```

Comment:
binding function in `render` is basically equal to `() => this.setSuggestionsListClicked()` so can we use arrow here?